packages/ttd-cli.vm/ttd-cli.vm.nuspec

```
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>ttd-cli.vm</id>
+    <version>1.11.319</version>
```

[nitpick] It look like the tool version uses the 4th segment, so strictly following [our documentation](https://github.com/mandiant/VM-Packages/wiki/Chocolatey-packages#non-metapackages), we should add the date as a forth segment. As the tool forth segment is zero, I have no strong opinion here.
```suggestion
    <version>1.11.319.20240614</version>
```

---

Done.